### PR TITLE
add simple pattern split tokenizer docs

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
add simple pattern split tokenizer docs

### Issues Resolved
Part of https://github.com/opensearch-project/documentation-website/issues/1483 addressed in this PR.

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _analyzers/tokenizers/simple-pattern-split.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+```
+
+## Parameters
+
+The `simple_pattern_split` tokenizer can be configured with the following parameter.
+
+Parameter | Required/Optional | Data type | Description
+:--- | :--- | :--- | :--- 
+`pattern` | Optional | String | The pattern used to split text into tokens specified using a [Lucene regular expression](https://lucene.apache.org/core/9_10_0/core/org/apache/lucene/util/automaton/RegExp.html). Default is an empty string, which does returns the input text unsplit.
```
Comment (kolchfa-aws) [inline _analyzers/tokenizers/simple-pattern-split.md:105]:
```suggestion
`pattern` | Optional | String | The pattern used to split text into tokens specified using a [Lucene regular expression](https://lucene.apache.org/core/9_10_0/core/org/apache/lucene/util/automaton/RegExp.html). Default is an empty string, which returns the input text as one token. 
```

File: _analyzers/tokenizers/simple-pattern-split.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
+layout: default
+title: Simple pattern split
+parent: Tokenizers
+nav_order: 120
+---
+
+# Simple pattern split tokenizer
+
+The `simple_pattern_split` tokenizer uses a regular expression to split text into tokens. The regular expression defines the pattern used to determine where to split the text. Any matching pattern in the text is used as a delimiter, and the text between delimiters becomes a token. Use this tokenizer when you want to to define delimiters and tokenize the rest of the text based on a pattern.
```
Comment (natebower) [inline _analyzers/tokenizers/simple-pattern-split.md:10]:
```suggestion
The `simple_pattern_split` tokenizer uses a regular expression to split text into tokens. The regular expression defines the pattern used to determine where to split the text. Any matching pattern in the text is used as a delimiter, and the text between delimiters becomes a token. Use this tokenizer when you want to define delimiters and tokenize the rest of the text based on a pattern.
```

File: _analyzers/tokenizers/simple-pattern-split.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+The tokenizer uses the matched parts of the input text (based on the regular expression) only as delimiters or boundaries to split the text into terms. The matched portions are not included as part of the resulting terms. For example, if the tokenizer is configured to split text on dot characters (`.`), and the input text is `one.two.three`, the generated terms are `one`, `two`, and `three`. The dot characters themselves are not included in the resulting terms.
```
Comment (natebower) [inline _analyzers/tokenizers/simple-pattern-split.md:12]:
```suggestion
The tokenizer uses the matched parts of the input text (based on the regular expression) only as delimiters or boundaries to split the text into terms. The matched portions are not included in the resulting terms. For example, if the tokenizer is configured to split text at dot characters (`.`) and the input text is `one.two.three`, then the generated terms are `one`, `two`, and `three`. The dot characters themselves are not included in the resulting terms.
```

File: _analyzers/tokenizers/simple-pattern-split.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+## Example usage
+
+The following example request creates a new index named `my_index` and configures an analyzer with a `simple_pattern_split` tokenizer. The tokenizer is configured to split text on hyphens:
```
Comment (natebower) [inline _analyzers/tokenizers/simple-pattern-split.md:16]:
```suggestion
The following example request creates a new index named `my_index` and configures an analyzer with a `simple_pattern_split` tokenizer. The tokenizer is configured to split text at hyphens:
```

File: _analyzers/tokenizers/simple-pattern-split.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+}
+```
+
+## Parameters
+
+The `simple_pattern_split` tokenizer can be configured with the following parameter.
+
+Parameter | Required/Optional | Data type | Description
+:--- | :--- | :--- | :--- 
+`pattern` | Optional | String | The pattern used to split text into tokens specified using a [Lucene regular expression](https://lucene.apache.org/core/9_10_0/core/org/apache/lucene/util/automaton/RegExp.html). Default is an empty string, which returns the input text as one token.
```
Comment (natebower) [inline _analyzers/tokenizers/simple-pattern-split.md:105]: If the tokens are specified, then fine as is. If, however, the pattern is specified, then there should be a comma after "tokens".

**Review by natebower — COMMENTED** [review]
@kolchfa-aws @AntonEliatra Editorial review. Thanks!

File: _analyzers/tokenizers/simple-pattern-split.md (hunk 5 above)
Comment (kolchfa-aws) [inline _analyzers/tokenizers/simple-pattern-split.md:105]:
```suggestion
`pattern` | Optional | String | The pattern used to split text into tokens, specified using a [Lucene regular expression](https://lucene.apache.org/core/9_10_0/core/org/apache/lucene/util/automaton/RegExp.html). Default is an empty string, which returns the input text as one token. 
```

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**